cmd/teams.go

```diff
@@ -193,6 +200,58 @@ func inviteToTeamCmd(cliCtx *cli.Context) error {
 	return nil
 }
 
+func membersTeamCmd(cliCtx *cli.Context) error {
+	ctx := context.Background()
+
+	teamID, err := validateTeamID(cliCtx)
+	if err != nil {
+		return err
+	}
+	if teamID == nil {
+		return cli.NewExitError("Can't view members for a non-team. Use `manifold switch` to select a team.", -1)
+	}
+
+	client, err := api.New(api.Identity)
+	if err != nil {
+		return err
+	}
+
+	prompts.SpinStart("Fetching Team Members")
+	members, err := clients.FetchTeamMembers(ctx, teamID.String(), client.Identity)
+	prompts.SpinStop()
+	if err != nil {
+		return cli.NewExitError(fmt.Sprintf("Failed to fetch list of teams: %s", err), -1)
+	}
+
+	prompts.SpinStart("Fetching Invites")
+	invites, err := clients.FetchInvites(ctx, teamID.String(), client.Identity)
+	prompts.SpinStop()
+	if err != nil {
+		return cli.NewExitError(fmt.Sprintf("Failed to fetch list of invites: %s", err), -1)
+	}
+
+	fmt.Printf("%d members and %d invites\n", len(members), len(invites))
+	fmt.Println("Use `manifold switch` to change to a different team")
+	fmt.Println()
+
+	w := ansiterm.NewTabWriter(os.Stdout, 0, 0, 8, ' ', 0)
+
+	w.SetStyle(ansiterm.Bold)
+	w.SetForeground(ansiterm.Gray)
+	fmt.Fprintln(w, "Name\tEmail\tRole\tStatus")
+	w.ClearStyle(ansiterm.Bold)
+	w.Reset()
+	for _, m := range members {
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\n", m.Name, m.Email, m.Role, "active")
+	}
+	w.SetStyle(ansiterm.Faint)
```

Comment:
I didn't realise there was a faint style already define. In most places we are using grey colour instead of faint. Maybe we can fix that later and add to the style guide.